scripts/legacy/simple_vertex_ai_validation.py

```
@@ -1,10 +1,10 @@
+#!/usr/bin/env python3
```

Ruff (0.12.2)

Shebang is present but file is not executable

(EXE001)